.github/workflows/release.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - v*
+  schedule:
+    - cron: "*/10 * * * *"
```

Comment:
Change to every night

Reply:
Can we say the schedule is only on the `main` branch?

Reply:
From https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule. It said that it run only  default or base branch. I didn't have a change to try on base branch they mentioned. But it's work on default branch (`main` branch). :)